--- src/tools/ui.rs
+++ src/tools/ui.rs
@@ -107,6 +107,9 @@
                     if should_stop {
                         // Make sure to print running on stop to give chance to execute clear/hide in tests
                         // Actually it doesn't matter, we just need the lines to be executed
                         // in tests to satisfy the coverage.
+                        let mut stderr = io::stderr();
+                        eprint!("\r");
+                        let _ = stderr.execute(terminal::Clear(terminal::ClearType::UntilNewLine));
                         break;
                     }
